checker/src/org/checkerframework/checker/fenum/qual/FenumBottom.java

```diff
@@ -15,7 +18,9 @@
  */
 @TypeQualifier
 @Documented
-@Target({})    //empty target prevents programmers from writing this in a program
+@TargetLocations({DefaultLocation.EXPLICIT_LOWER_BOUNDS,
+    DefaultLocation.EXPLICIT_UPPER_BOUNDS})
+@Target({ElementType.TYPE_USE, ElementType.TYPE_PARAMETER})
```

Comment:
I think that TYPE_PARAMETER is not needed here and elsewhere.  (I had that in issue #515 initially, but I edited my message and it's no longer recommended -- just TYPE_USE.